.circleci/config.yml

```diff
@@ -1146,6 +1148,7 @@ jobs:
     environment:
       <<: *base_osx_env
       CMAKE_BUILD_TYPE: Release
+      CMAKE_OPTIONS: -DCMAKE_OSX_ARCHITECTURES:STRING=x86_64;arm64
```

Comment:
There are quite a few warnings like these in the log:
```
ld: warning: ignoring duplicate libraries: '../libevmasm/libevmasm.a', '../libsmtutil/libsmtutil.a', '../libsolidity/libsolidity.a', '../libsolutil/libsolutil.a', '../libyul/libyul.a', '/usr/local/lib/libboost_filesystem.a'
ld: warning: ignoring duplicate libraries: '../libevmasm/libevmasm.a', '../libsmtutil/libsmtutil.a', '../libsolidity/libsolidity.a', '../libsolutil/libsolutil.a', '../libyul/libyul.a', '/usr/local/lib/libboost_filesystem.a'
```

What's happening there?

Reply:
Hmm.. the truth is that I don't exactly know what is causing that. But I think it is not critical.

However, I see these warnings always when I run a build on macOS. This PR is not responsible for these warnings.  

```
[100%] Building CXX object test/CMakeFiles/soltest.dir/yulPhaser/SimulationRNG.cpp.o
[100%] Linking CXX executable soltest
ld: warning: ignoring duplicate libraries: '../libevmasm/libevmasm.a', '../libsmtutil/libsmtutil.a', '../libsolidity/libsolidity.a', '../libsolutil/libsolutil.a', '../libyul/libyul.a', '/opt/homebrew/lib/libboost_filesystem-mt.a'
[100%] Built target soltest
```

Not sure weather it makes sense to investigate this further. You don't see similar warnings using `gcc` on linux?

Reply:
No, I haven't seen them so far in linux builds. There also aren't any in the output from [`b_osx` run on `develop`](https://app.circleci.com/pipelines/github/ethereum/solidity/32627/workflows/70749d51-5b9f-49b8-a54b-0e3ac566b16d).

But ok, I guess if it was broken to the point of either not linking one of these libs or linking the one for the wrong platform, the binary would not work at all. So we probably don't have to worry about it. I think it's still worth trying to google around and see what may be causing it, but it's not critical. It's fine to check this after release.